joomla-projects/privacy-framework
# Support export user custom fields data

Pull Request for Issue # .

### Summary of Changes
This PR is a small addition to https://github.com/joomla-projects/privacy-framework/pull/53 to allow exporting user custom fields data for export request.

I don't know what fields need to be included for each item, so I added user_id, field_id, field_title and field_value for now. If we need more data, or have to change it, let me know and I will make the changes.

### Testing Instructions
1. Create user custom fields
2. Apply both the PRs
3. Make an export request for an user, try to export data for that request, make sure user custom fields data is exported properly
